scripts/postinstall.js

```diff
@@ -0,0 +1,47 @@
+import { Buffer } from 'buffer';
+import fetch from 'cross-fetch';
+import { execa } from 'execa';
+import { existsSync } from 'fs';
+import { writeFile } from 'fs/promises';
+import { homedir } from 'os';
+import { join } from 'path';
+
+(async () => {
+    // Because this is loaded as a module, __dirname is not available (https://github.com/nodejs/help/issues/2907)
```

Comment:
But isn't that quite a big problem? I'm pretty sure this means that the venv will be created in your cwd.

We should use [`import.meta.url`](https://nodejs.org/api/esm.html#importmetaurl) or something like <https://www.npmjs.com/package/cross-dirname>.

Reply:
As you're doing in `index.ts`…

Reply:
I forgot to document this, but scripts are always run from the project root (https://docs.npmjs.com/cli/v9/using-npm/scripts#best-practices).